packages/core/src/app/checkout/mapToCheckoutProps.ts

```diff
@@ -40,8 +44,10 @@ export default function mapToCheckoutProps(
         isGuestEnabled,
         isLoadingCheckout: statuses.isLoadingCheckout(),
         isPending: statuses.isPending(),
+        isPriceHiddenFromGuests,
```

Comment:
Can I know how did you set this prop? Think it should be something like `isPriceHiddenFromGuests: features['XXXX']`?

Reply:
Hey @bc-peng on line 26-28, I destructure the "hidePriceFromGuests" value returned from the `checkout-settings` endpoint and rename it to `isPriceHiddenFromGuests`. That prop is set directly from the checkout-settings API response.

I expose this store config in BCApp to the checkout-settings endpoint here: https://github.com/bigcommerce/bigcommerce/blob/0097e6a8ffd964c973224b6ec66810ecdcee7ec4/lib/Services/Checkout/Helpers/CheckoutDataHelper.php#L808

Reply:
Thanks for the detailed explantation. All clear!